omod/src/test/java/org/openmrs/module/aijar/web/AppointmentControllerTest.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.aijar.web;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+public class AppointmentControllerTest extends RestControllerTestUtils {
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSet("org/openmrs/module/aijar/include/standardTestDataset.xml");
+        executeDataSet("org/openmrs/module/aijar/include/appointmentDataSet.xml");
+    }
+
+    @Test
+    public void shouldReturnPatientsWhenNoSpecifiedEncounterUuids() throws Exception {
```

Comment:
@reagan-meant add tests for a date which has no data (add a January 1, 1980) and a test for an encounter type which has does not exist so should return 0 patients

Reply:
Done